admin-dev/themes/new-theme/scss/pages/product/create_product_default_theme.scss

```
@@ -0,0 +1,41 @@
+
+#modal-create-product {
+  //modal loader
+  @import "~bootstrap/scss/mixins/border-radius";
+  @import "../../config/settings";
+  @import "~prestakit/scss/spinners";
+  @import "../../components/modal";
+  @import "create_product_modal";
+
+  .d-none {
+    /* stylelint-disable declaration-no-important */
+    display: none !important;
+    /* stylelint-enable declaration-no-important */
+  }
+
+  .modal-header {
```

I think we could customize this one a bit more to make it look like the one on the new theme:

```suggestion
  .modal-dialog {
    top: 50%;
    -webkit-transform: translateY(-50%);
    transform: translateY(-50%);
    width: 90%;
    max-width: 810px;
  }

  .modal-header {
      border-bottom: none;
```